locale.c

```diff
@@ -2748,6 +2748,20 @@ S_save_to_buffer(const char * string, const char **buf, Size_t *buf_size)
                           *buf));
             }
 
+#    ifdef DEBUGGING
```

Comment:
FWIW, I would prefer we were consistent about whether and when we outdent ifdefs, personally i would not outdent this, and not outdenting seems to be the standard (Although i guess some of these hits could be from code where no outdent was required).

```
~/git_tree/perl$ git grep -P '^#\s+ifdef DEBUGGING' | wc -l
25
~/git_tree/perl$ git grep -P '^#ifdef DEBUGGING' | wc -l
208
```

Reply:
From perlhack:

>   4-wide indents for code, 2-wide indents for nested CPP "#define"s,

I think the wording on that is wrong, it meant to include any nested CPP directives.  Otherwise it doesn't make sense

It doesn't much matter on files that don't have much nesting.  But  locale.c has a lot due to the vagaries of various platforms.  When I changed that to have indenting, it made it a lot clearer

> § git grep -P '^#\s+(if|el|endif)' | wc -l 
7290
git grep -P '^#(if|el|endif)' | wc -l
17141